src/Microsoft.Azure.Relay/Common/InputQueue.cs

```diff
@@ -568,20 +559,11 @@ public Item(Exception exception, Action dequeuedCallback)
                 this.dequeuedCallback = dequeuedCallback;
             }
 
-            public Action DequeuedCallback
-            {
-                get { return this.dequeuedCallback; }
-            }
+            public Action DequeuedCallback => this.dequeuedCallback;
```

Comment:
As long as you're touching these could we eliminate the fields altogether and just use the readonly auto-property syntax?
``` C#
public Action DequeuedCallback { get; }

public Exception Exception { get; }

public Exception Exception { get; }
```

Reply:
Sure Personally  I prefer that sintaxis 

`public Action DequeuedCallback { get; }` if is readonly and 
`public Action DequeuedCallback { get; private set; }` if need to be modified outside the ctor

Reply:
Hopefully none of those fields/properties are modified outside the .ctor.